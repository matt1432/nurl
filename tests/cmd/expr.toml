args = ["--expr", "(import <nixpkgs> { }).emptyFile"]
