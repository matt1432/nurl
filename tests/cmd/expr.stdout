sha256-d6xi4mKdjkX2JFicDIv5niSzpyI0m/Hnm8GGAIU04kY=